src/sidebar/helpers/mention-suggestions.ts

```diff
@@ -0,0 +1,44 @@
+import type { FocusedGroupMembers } from '../store/modules/groups';
+
+export type UserItem = {
+  username: string;
+  displayName: string | null;
+};
+
+export type UsersForMentions =
+  | { status: 'loading' }
+  | { status: 'loaded'; users: UserItem[] };
+
+/**
+ * Merge, deduplicate and sort a list of users to be used for mention suggestions.
+ * The list includes both users who already annotated the document, and members
+ * of the currently focused group.
+ *
+ * We won't return any user if the group members are being loaded, preventing a
+ * mix of some already-known users and a loading indicator to be shown at the
+ * same time.
+ */
+export function resolveUsersForMentions(
```

Comment:
```suggestion
export function combineUsersForMentions(
```

To me "resolve" suggests an operation like resolving a relative path to an absolute one or something. Words like _combine_, _merge_ or _collate_ would make the purpose of this function more obvious.